### PR TITLE
Update the URL for the preservation_catalog

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

The preservation_catalog changed it's url. 

## Was the usage documentation (e.g. README, DevOpsDocs, wiki, queue specific README) updated?
n/a

### CODE REVIEW

File: lib/robots/sdr_repo/preservation_ingest/update_catalog.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require 'faraday'
 require 'retries'
-
+require 'byebug'
```
Comment (justinlittman) [inline lib/robots/sdr_repo/preservation_ingest/update_catalog.rb]: ?

Comment (peetucket) [reply lib/robots/sdr_repo/preservation_ingest/update_catalog.rb]: assumed this is useful for debugging purposes during development, but yes, not sure why its in the class now that you mention it

Comment (jcoyne) [reply lib/robots/sdr_repo/preservation_ingest/update_catalog.rb]: fixed.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@
 require 'webmock/rspec'
 require 'simplecov'
 require 'coveralls'
+require 'byebug'
```
Comment (aaron-collier) [inline spec/spec_helper.rb]: Do we care about byebug in spec_helper?

Comment (jcoyne) [reply spec/spec_helper.rb]: This is the proper place for it.

Comment (ndushay) [reply spec/spec_helper.rb]: no.

**Pull request merged**